### ISSUE TITLE
Prints never finish

### ISSUE THREAD

**Issue by sybarite12** [Toybox-Labs/support-page#5]
Maybe 1 out of every 8 prints finish, instead receiving message: "your toybox had to cancel, please try again". Wasting LOTS of printer food and irritating impatient kids. :(

**Comment by schlaegerz** [comment]
The best way to fix this is to try a factory reset, 
 https://drive.google.com/a/make.toys/file/d/1KRAxDXav4iAAAIY4BkS35FEW4sLndv7b/view?usp=drive_web

**Comment by sybarite12** [comment]
I emailed you a very long time ago. I have fixed the problem myself but 3t
his does not give you high ranks for customer service.

**Comment by ToyboxLabs** [comment]
Yes very sorry about that, we did not intend for people to find this page to request help, sorry for the confusion.
